test/test_arraycontext.py

```diff
@@ -868,11 +868,12 @@ def test_container_norm(actx_factory, ord):
     actx = actx_factory()
 
     from pytools.obj_array import make_obj_array
-    c = MyContainer(name="hey", mass=1, momentum=make_obj_array([2, 3]), enthalpy=5)
+    c = MyContainer(name="hey",
+            mass=1.0, momentum=make_obj_array([2.0, 3.0]), enthalpy=5.0)
```

Comment:
Not quite sure what happened here, but `pyopencl` didn't seem to want to `pow` some combination of integers and floats in the norm.